File: RS.VueClient/src/Models/WebAPI/LoginResultModel.ts
import type { SessionModel } from "./SessionModel";

/**
 * 登录结果
 */
export class LoginResultModel {
  /**
   * 新的会话Model
   */
  public SessionModel: SessionModel | null = null;

  /**
   * 返回昵称
   */
  public NickName: string | null = null;

  /**
 * 返回用户的头像
 */
  public UserImgUrl: ArrayBuffer | null = null;

}    

File: RS.Models/LoginResultModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RS.Models
{

    /// <summary>
    /// 登录结果
    /// </summary>
    public class LoginResultModel
    {
        /// <summary>
        /// 新的会话Model
        /// </summary>
        public SessionModel? SessionModel { get; set; }

        /// <summary>
        /// 返回昵称
        /// </summary>
        public string? NickName { get; set; }

        /// <summary>
        /// 返回用户的头像
        /// </summary>
        public byte[]? UserImgUrl { get; set; }


    }
}
